src/components/GlobalStyle.js

```diff
@@ -3,6 +3,7 @@ import reset from 'styled-reset'
 
 export const BREAKPOINT_SMALL = '500px'
 export const BREAKPOINT_MEDIUM = '768px'
+const LINK_COLOR = '#008AFF'
```

Comment:
Just as an idea. Not asking for this to be changed now but I think it would be good to extract more of the global styles up here. As well possibly to think about using CSS Variables: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
I kinda like this pattern of exporting global variables since we're using the CSSinJS approach.  Do you know if people use css variables with styled components? I'm unsure on conventions. 🤷‍♂️

Reply:
They both do similar so I think it may be something to possibly look into. we can keep it as is now and if we feel things are getting cluttered, then we can look into other things to make it cleaner later on.